ownership-chain/runtime/src/precompiles/test.rs

```diff
@@ -24,8 +23,84 @@ fn ethereum_precompiled_addresses_are_precompile() {
 	assert!(is_precompile(hash(3)).unwrap());
 	assert!(is_precompile(hash(4)).unwrap());
 	assert!(is_precompile(hash(5)).unwrap());
+	assert!(!is_precompile(hash(6)).unwrap());
 	assert!(!is_precompile(hash(1026)).unwrap());
 	assert!(is_precompile(hash(1027)).unwrap());
 	assert!(is_precompile(H160::from_str("0xfffffffffffffffffffffffe0000000000000005").unwrap())
 		.unwrap());
 }
+
+#[test]
+/// Test to ensure that delegate calls to addresses that are not precompiles are recognized
```

Comment:
comment above macro attribute looks cleaner and this is what we do in the rest of the codebase. So would suggest to re-order.

Reply:
sgtm